### PR TITLE
Show notifications even when Element is focused

### PR DESCRIPTION
**Pull request by sumnerevans**
This makes Element show notifications for messages even when it is the
focused application.

Before this change, Element would not show any notifications while
focused on the app. This causes confusion to end-users, especially if
the room where the notification came in is in a different space.

```
Signed-off-by: Sumner Evans <sumner@beeper.com>
```

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Show notifications even when Element is focused ([\#8590](https://github.com/matrix-org/matrix-react-sdk/pull/8590)). Contributed by @sumnerevans.

### CODE REVIEW

File: src/Notifier.ts [hunk 1]
```diff
@@ -95,9 +95,6 @@ export const Notifier = {
         if (!plaf.supportsNotifications() || !plaf.maySendNotifications()) {
             return;
         }
-        if (global.document.hasFocus()) {
```
Comment (turt2live) [inline src/Notifier.ts:98]: is there a check to prevent notifications for the currently active room somewhere?

**Review by turt2live — COMMENTED** [review]
Thanks for the PR! This also requires product review, so will request their opinion.

File: src/Notifier.ts (hunk 1 above)
Comment (sumnerevans) [reply src/Notifier.ts:98]: I think that is what is happening here: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/Notifier.ts#L404-L410

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
Could we please not show notifications for the currently focused room?

**Comment by turt2live** [comment]
@SimonBrandner https://github.com/matrix-org/matrix-react-sdk/pull/8590#discussion_r872891813 ?

**Review by SimonBrandner — APPROVED** [review]
LGTM codewise!

**Review by jakewb-b — APPROVED** [review]
Happy with this from a product point of view. Nice enhancement.

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**